lms/static/js/verify_student/views/step_view.js

```diff
@@ -101,4 +103,4 @@
 
     });
 
- })( jQuery, _, _.str, Backbone, gettext );
+ })( jQuery, _, Backbone, gettext );
```

Comment:
If there's an entry in the require config for "_.str", then it should be defined by the time that this runs. Also, we have a bigger problem if this file executes before requirejs has initialized and loaded all of its dependencies. Could you explain a little more? I'm uncomfortable with merging code that just happens to load its dependencies in time rather than being guaranteed to do so.